packages/manager/modules/nutanix/src/dashboard/nodes/node/ipmi/routing.js

```diff
@@ -0,0 +1,9 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('nutanix.dashboard.nodes.node.ipmi', {
+    url: '/ipmi',
+    component: 'nutanixNodeIpmi',
+    resolve: {
+      breadcrumb: /* @ngInject */ () => 'IPMI',
```

Comment:
`IPMI` should probably come from translations?

Reply:
NO, its a technical word. No need to translate.